src/config.c

```diff
@@ -331,6 +332,16 @@ static int parse_weights(const char *input, int weights[MAX_NODES])
 	return i;
 }
 
+/* returns TICKET_MODE_AUTO if failed to parse the ticket mode. */
+static ticket_mode_e retrieve_ticket_mode(const char *input)
+{
+	if ((strcmp(input, "manual") == 0) ||
+	    (strcmp(input, "MANUAL") == 0)) {
```

Comment:
strcasecmp? Actually, I cannot recall if other stuff in the configuration file considers the case at all. Best to try to be consistent with what's already there.

Reply:
sure, will use strcasecmp instead